### PR TITLE
Tags and Tag Components Documentation and Examples

### PR DESCRIPTION
**Pull request by phuson**
Adds documentation and examples for how to use Tags and Tag components in this PR: https://github.com/grommet/grommet/pull/295

### CODE REVIEW

File: src/develop/components/TagsDoc.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+          <h3>Default</h3>
+          <Box className="example">
+            <Tags>
+              <Tag label="First" />
+              <Tag label="Second" />
+              <Tag label="Third" />
+            </Tags>
+          </Box>
+          <pre><code className="html hljs xml">
+            {"<Tags>\n  <Tag label='First' />\n  ...\n</Tags>"}
```
Comment (alansouzati) [inline src/develop/components/TagsDoc.js]: Can we use the amazing `jsx-to-string` library :100: ahaha

See this for an example on how to change this doc page: https://github.com/grommet/grommet-docs/blob/master/src/develop/components/AnchorDoc.js

**Comment by alansouzati** [comment]
screen shot 2016-01-14 at 12 04 30 am

This is how it looks in case you are reviewing this pull request. I like it.

@ericsoderberghp and @L0ZZI would love your insights here.

**Comment by alansouzati** [comment]
If we take a look here: http://www.grommet.io/docs/develop/anchor.

This tag is somehow close to a colored context. I would follow the same standard for hovering. This means that I would replace the original blue underline by the same color with underline.

File: src/develop/components/TagsDoc.js (hunk 1 above)
Comment (phuson) [inline src/develop/components/TagsDoc.js]: :+1: on the `jsx-to-string` library :smile:

**Comment by alansouzati** [comment]
thanks for your contribution.

**Comment by L0ZZI** [comment]
Hey Son, It should look like this:
screenshot 2016-01-22 14 06 54

It's based on button (secondary) but with the stroke at like 0.3 alpha.

**Comment by phuson** [comment]
@L0ZZI Ahh, that looks new. How about the hover state for these, what does that look like?

**Comment by L0ZZI** [comment]
it turns into secondary button.

**Comment by L0ZZI** [comment]
uses the same style it's just more feint.

**Comment by phuson** [comment]
@L0ZZI Just making sure I understand this correctly: we want the Tag component's styling to be exactly the same as the **Secondary** button on the button docs (http://www.grommet.io/docs/develop/button), but with a grey transparent stroke. If so, the current Grommet button implementation has rounded corners, based on the attached design screenshot, we won't have that for the Tag component, right?

OR

Is the attached screenshot above just describe the hover state of the Tag component? :smile:

**Comment by L0ZZI** [comment]
I think you are correct @phuson. I would keep the rounded corner param consistent across themes. If the HPE theme **has no rounded corners** on the button, that should be carried over to the tag element (http://www.grommet.io/docs/hpe/develop/button). In the Grommet Open Source theme the same should be true, meaning the tags would be rounded.

As for the the color and the state, the stroke and text would be .3 alpha of the Secondary button active state, and then the hover/press state would match the Secondary button active state. There should be no disabled state for the tag (it shouldn't be shown).

**Pull request merged**